lrauv_system_tests/test/simulation/test_spawn.cc

```diff
@@ -149,16 +149,6 @@ TEST(VehicleSpawnTest, Spawn)
 
   std::vector<std::string> topics;
   node.TopicList(topics);
-  EXPECT_TRUE(std::any_of(
-      topics.begin(), topics.end(),
-      [](const std::string &s) {
-        return s == "/comms/external/201/tx";
-      }));
-  EXPECT_TRUE(std::any_of(
-      topics.begin(), topics.end(),
-      [](const std::string &s) {
-        return s == "/comms/external/202/tx";
-      }));
```

Comment:
@adityapande-1995 hmm, mind to explain why we are removing?

Reply:
This is again a remnant of ``main`` : The old comms used these topics to transmit and receive msgs, but now we have the broker which does the same job, and these topics are no longer created. Instead, we have ``/broker/msgs/`` and ``/1/rx``, both of which are handled by the upstream ``AcousticComms`` and ``Endpoint`` plugins. These existeance of those topics is tested in gz-sim, so I removed that check from here.